gdal/gcore/gdaldataset.cpp

```diff
@@ -4578,6 +4578,30 @@ OGRLayerH GDALDatasetExecuteSQL( GDALDatasetH hDS,
             pszDialect));
 }
 
+
+/************************************************************************/
+/*                        GDALDatasetAbortSQL()                       */
+/************************************************************************/
+
+/**
+ \brief Abort any SQL statement running in the data store.
+
+ This method is the same as the C++ method GDALDataset::AbortSQL()
+
+ @since GDAL 3.1.4
```

Comment:
We can't add this to a stable release, especially since it involves adding a new virtual method, which changes the C++ ABI that must remain stable in the 3.1 series.

```suggestion
 @since GDAL 3.2.0
```